pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -38,7 +38,10 @@
 
 class ModelCheckpoint(Callback):
     r"""
-    Save the model after every epoch by monitoring a quantity.
+    Save the model after every epoch by monitoring a quantity. Every metric logged with
```

Comment:
Is this an old version of Lightning? the "after every epoch" isn't true - there are more options being added for greater flexibility. See https://github.com/PyTorchLightning/pytorch-lightning/blob/19e67d18c472c3a03dec4dd9bfcef031e9ca8719/pytorch_lightning/callbacks/model_checkpoint.py#L96-L107

Reply:
Actually it was like that in the documentation, that's why I didn't change the `after every epoch` part. But you are right. What do you think about this before the part that I added:

Save the model after every ``every_n_train_steps`` training steps or every ``every_n_val_epochs`` validation epochs by monitoring a quantity. These options are mutually exclusive.

Reply:
I would not mention the arguments before they are introduced later on. 
How about:

Save the model periodically by monitoring a quantity. ...